test/parallel/test-net-reconnect-error.js

```diff
@@ -10,8 +10,7 @@ let disconnect_count = 0;
 const c = net.createConnection(common.PORT);
 
 c.on('connect', function() {
-  console.error('CLIENT connected');
-  assert.ok(false);
+  common.fail('client should not have connected');
```

Comment:
`common.fail()` expects a message for the first argument. If you do `c.on('connect', common.fail);`, you end up with `AssertionError: null undefined null` which is not nearly as useful as `AssertionError: client should not have connected`.